src/modules/70-pipeline/components/PipelineSteps/Steps/JiraCreate/JiraFieldsRenderer.tsx

```diff
@@ -79,24 +111,61 @@ function GetMappedFieldComponent({
         placeholder={selectedField.name}
         className={css.md}
         multiTextInputProps={{
+          allowableTypes: allowableTypes,
           expressions
         }}
       />
     )
   } else if (shouldShowMultiSelectField(selectedField)) {
-    return (
-      <FormInput.MultiSelectTypeInput
-        selectItems={setAllowedValuesOptions(selectedField.allowedValues)}
-        label={selectedField.name}
-        disabled={isApprovalStepFieldDisabled(props.readonly)}
-        name={formikFieldPath}
-        placeholder={selectedField.name}
-        className={cx(css.multiSelect, css.md)}
-        multiSelectTypeInputProps={{
-          expressions
-        }}
-      />
-    )
+    {
+      return props.deploymentMode ? (
+        <div className={css.btnPosition}>
+          <Label style={{ color: Color.GREY_900 }}>{selectedField?.name}</Label>
+          <MultiSelectTypeInput
+            width={390}
+            disabled={isApprovalStepFieldDisabled(props.readonly)}
+            name={formikFieldPath}
+            placeholder={selectedField.name}
+            className={cx(css.multiSelect, css.md, {
+              [css.formError]: !isNil(get(props.formik?.errors, formikFieldPath))
+            })}
+            multiSelectProps={{
+              items: setAllowedValuesOptions(selectedField?.allowedValues)
+            }}
+            expressions={expressions}
+            allowableTypes={allowableTypes}
+            onChange={items => {
+              const valueArr = [] as string[]
+              ;(items as any)?.forEach((opt: any) => {
+                if (opt?.value) valueArr.push(opt?.value)
+              })
+              valueArr?.join(',')
+              props.formik.setFieldValue(formikFieldPath, valueArr.toString())
```

Comment:
`valueArr?.join(',')` can be removed `valueArr.toString()` will join using `,`
```suggestion
              props.formik.setFieldValue(formikFieldPath, valueArr.toString())
```